roles/pulp_webserver/tasks/apache.yml

```diff
@@ -6,25 +6,50 @@
         name: "{{ pulp_webserver_apache_package }}"
         state: present
 
+    - name: Change unix socket to ip
+      set_fact:
+        pulp_api_bind: >-
```

Comment:
Please avoid the use of `set_fact`.
You can define a `__pulp_webserver_apache_api_bind` variable in `vars/main.yml` of this role using the `ternary` filter.

`__pulp_webserver_apache_api_bind: "{{ pulp_api_bind.startswith('unix') | ternary(pulp_api_bind + '|http://' + pulp_api_apache_url, 'http://' + pulp_api_bind ) }}"`
should work.